### DISCUSSION TITLE
Enable the upload option in CI Hub

### DISCUSSION THREAD

**Discussion by PGS-gh** [pimcore/pimcore#9927] [Q&A / Support]
**Basic Configuration**
I configured Microsoft Word and PPT with CI Hub.
After successful login and adding Pimcore connection, I can drag n drop the assets and metadata related to it in the word or the ppt. Also, able to download the assets as well.

**Requirement**
I want to upload my changes like that of word, ppt or images directly into Pimcore through CI Hub.

**Issue**
There is an upload option available for the file-types. But it is been disabled.

_Is there a way to enable the upload button, so that one can directly upload the changes in the Pimcore?
If yes, then where can one do so?_

**Comment by labudzinski** [comment]
@PGS-gh https://github.com/BrandOriented/PimcoreCiHubConnector
